lib/webrtc/call.js

```diff
@@ -74,6 +74,12 @@ MatrixCall.ERR_LOCAL_OFFER_FAILED = "local_offer_failed";
  */
 MatrixCall.ERR_NO_USER_MEDIA = "no_user_media";
 
+/**
+ * Lookup from opaque queue ID to a promise for media element operations that
+ * need to be serialised into a given queue
+ */
+MatrixCall.mediaPromises = {};
```

Comment:
Also, we're using this as a dictionary where an external entity (the caller) can set arbitrary keys. We should make this a proper map then using `Object.create(null)` (an object without a prototype) else you can get [fun bugs](https://github.com/matrix-org/matrix-appservice-irc/issues/152) if people decide to use "hasOwnProperty" as a queue ID. It's unlikely to happen in this particular case, but given it's a direct replacement and everything else remains the same, it's trivial to do.